src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -99,5 +99,13 @@ public override bool IsExplicitMatch(ITest test)
 
         //    return false;
         //}	
+
+        public override TNode AddToXml(TNode parentNode, bool recursive)
+        {
+            TNode result = parentNode.AddElement("not");
+            if (recursive)
+                BaseFilter.AddToXml(result, true);
+            return result;
+        }
     }
```

Comment:
I am concerned that this will result in the "not" being lost in the output?  I perhaps am confused?